src/core/reducers/categories.js

```diff
@@ -51,6 +52,16 @@ export default function categories(state = initialState, action) {
         });
       });
 
+      // Android doesn't have any theme categories but because all lightweight
+      // themes (personas) are installable on Firefox Desktop and Android
+      // we share categories and themes across clientApps.
+      // See: https://github.com/mozilla/addons-frontend/issues/2170
+      //
+      // This can be removed once
+      // https://github.com/mozilla/addons-server/issues/4766 is fixed.
+      categoryList.android[ADDON_TYPE_THEME] = categoryList
```

Comment:
I know it's not part of this patch but could you log a warning [when we skip an app category](https://github.com/mozilla/addons-frontend/blob/f333d5eb389a017de2641692cf260c5bfc0e6476/src/core/reducers/categories.js#L35)? That could be very confusing if we have incorrect values configured for valid apps.